api/src/test/java/org/openmrs/module/registrationcore/RegistrationCoreUtilTest.java

```diff
@@ -1,55 +1,95 @@
 package org.openmrs.module.registrationcore;
 
-import org.joda.time.DateTime;
-import org.junit.Test;
-
 import static org.hamcrest.core.Is.is;
 import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
-public class RegistrationCoreUtilTest {
-
-    @Test
-    public void calculateBirthdateFromAge_shouldCalculateProperBirthdateWhenYearsSpecified(){
-        DateTime ageOnDate= new DateTime(2015, 04, 03, 0, 0, 0);
-        DateTime result = new DateTime(RegistrationCoreUtil.calculateBirthdateFromAge(10, null, null, ageOnDate.toDate()));
-        assertThat(result.getYear(), is(2005));
-        assertThat(result.getMonthOfYear(), is(1));
-        assertThat(result.getDayOfMonth(), is(1));
-    }
-
-    @Test
-    public void calculateBirthdateFromAge_shouldCalculateProperBirthdateWhenYearAndMonthSpecified(){
```

Comment:
It has hard to tell which of these tests actually changed. Can you add you new tests without touching the rest of the file?